src/app/account-app/credit-cards/credit-cards.component.ts

```diff
@@ -76,9 +79,26 @@ export class CreditCardsComponent implements OnInit {
         this.refreshCards();
     }
 
+    addCard() {
+        this.rmmapi.addNewCard().subscribe(
+            res => {
+                const dialogRef = this.dialog.open(StripeAddCardDialogComponent, { data: { clientSecret: res.client_secret } });
+                dialogRef.afterClosed().subscribe(cardAdded => {
+                    if (cardAdded) {
+                        // not pretty, but forces the progress spinner to show up again,
+                        // and it's better than looking like nothing happened
+                        this.creditCards = new ReplaySubject<any>(1);
+                        this.refreshCards();
+                    }
+                });
+            },
+            _err => this.snackbar.open('Failed to add a new credit card', 'Okay'),
```

Comment:
What sort of error is this one? Can we get any reasons why it errored?

Reply:
Mebbe use the "contact support" one? (tho unless support get some sort of error log, not all that helpful either)

Reply:
Yeah, we don't have a snackbar form of "contact support" yet – maybe this is a good time to introduce it, actually.

Reply:
Fixed now in cde0ae33aff779f486420f4dfdc3fe3fb2fa0124